audio_service/example/lib/main.dart

```diff
@@ -776,264 +779,192 @@ class MediaLibrary {
   static const albumsRootId = 'albums';
 
   final items = <String, List<MediaItem>>{
-    AudioService.browsableRootId: [
+    AudioService.browsableRootId: const [
       MediaItem(
         id: albumsRootId,
-        album: "",
-        title: "Albums",
+        album: '',
+        title: 'Albums',
         playable: false,
       ),
     ],
     albumsRootId: [
       MediaItem(
-        id: "https://s3.amazonaws.com/scifri-episodes/scifri20181123-episode.mp3",
-        album: "Science Friday",
-        title: "A Salute To Head-Scratching Science",
-        artist: "Science Friday and WNYC Studios",
-        duration: Duration(milliseconds: 5739820),
+        id: 'https://s3.amazonaws.com/scifri-episodes/scifri20181123-episode.mp3',
```

Comment:
Dart is really pushing for single quotes everywhere even though they created a language where we have a free choice. What do I do? I tend to use double quotes for English phrases because the apostrophe is a common character in English. I would hate to use single quotes by default for English phrases and then have change it to double quotes when by chance I happen to edit a word that happens to introduce an apostrophe.

I don't care too much about having a mix, e.g. I wouldn't object to using single quotes for non-English strings like string identifier values, map keys, etc.